public_html/wp-content/plugins/pattern-directory/includes/favorite.php

```diff
@@ -84,3 +84,81 @@ function get_favorites( $user = 0 ) {
 
 	return array_map( 'absint', $favorites );
 }
+
+/**
+ * Get the cached count of how many times this pattern has been favorited.
+ *
+ * @param int|WP_Post $post The pattern to check.
+ * @return integer
+ */
+function get_favorite_count( $post = 0 ) {
+	$post = get_block_pattern( $post );
+	if ( ! $post ) {
+		return false;
+	}
+
+	return absint( get_post_meta( $post->ID, META_KEY, true ) );
```

Comment:
Since this is using the same constant value for both a user meta key and a post meta key, I think it would be worth explaining that in a comment up where `META_KEY` is defined.